docs/services.md

```diff
@@ -1,10 +1,33 @@
-# Services we can provide
+# Roles and Services
+
+The panels below describe the various services and roles in a typical hackweek. Here is the legend for reviewing these roles:
 
 ````{panels}
+:column: col-4
+:card: border-2
 
+:::{admonition} Services we usually provide
```

Comment:
admonitions have awkward rendering on narrow web browsers, looks like you can just color the title of each panel differently rather than using an admonition see example here https://jupyterbook.org/intro.html#find-the-right-documentation-resources
```
:::{panels}
:container: +full-width
:column: col-lg-4 px-2 py-2
---
:header: bg-jb-one
**Get started**
^^^

**[](start/your-first-book.md)**: a step-by-step tutorial to get started.

**[](create-a-template-book)**: get started with a simple template book.

---
```

Reply:
maybe just `:container: +full-width` is what is needed? full docs on styling and layout options here https://sphinx-panels.readthedocs.io/en/latest/#card-layout

Reply:
Right, so you'll see I used that approach in `intro.md`:

```
:::{panels}
:container: +full-width
:column: col-lg-4 px-2 py-2
---
:header: bg-jb-one
**<span style='color:white'>What is a hackweek?</span>**
^^^
Learn more about the hackweek model at our [Hackweek Mission](mission) page.

Get to know [our team](team) of hackweek designers and supports.

Browse the [services](services) we can provide.
---
```

to get the purple panel background. However, within `services.md` I want to have drop down admonitions within a series of panels, to list out each role. That is were I can't seem to customize the admonition colors.